### ISSUE TITLE
In Python3 - license text is displayed as a byte type

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#275]
When selecting to view a license request from the license requests table, the text shows prepended with `b'` and the quotes are escaped.

The full display is below:

```
Test License [Please Ignore]

Fullname

Test License [Please Ignore]

License author name

Short Identifier

testid

Source/URL

http://source1.com

Osi Approved

true

Notes

None

Standard License Header

None

Text

b'

this doesn\'t match

'
Submission Date

Dec. 16, 2020, 11:12 p.m.
```

**Comment by goneall** [comment]
Resolved in PR #276

**Comment by goneall** [comment]
Resolved with PR #276
